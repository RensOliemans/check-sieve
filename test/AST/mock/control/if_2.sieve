if true {
    keep;
    stop;
} else {
    stop;
}