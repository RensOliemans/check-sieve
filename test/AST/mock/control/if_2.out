Mail Sieve
    Branch
        Condition
            True
        Block
            Command (keep)
            Command (stop)
        Block
            Command (stop)
